src/functions/questionnaire/list-questionnaire/handler.ts

```diff
@@ -0,0 +1,42 @@
+import type { APIGatewayProxyHandler } from "aws-lambda";
+import type QuestionnaireModel from "src/database/models/questionnaire";
+import { questionnaireService } from "src/database/services";
+import { middyfy } from "src/libs/lambda";
+
+/**
+ * Labmda for listing all questions from DynamoDB.
+ */
+
+const listQuestionsHandler: APIGatewayProxyHandler = async () => {
+  try {
+    const allQuestionaires: QuestionnaireModel[] =
+      await questionnaireService.listQuestionnaires();
+
+    const mappedQuestionaires = allQuestionaires.map((questionaire) => {
+      return {
+        id: questionaire.id,
+        title: questionaire.title,
+        description: questionaire.description,
+        options: questionaire.options,
+        tags: questionaire.tags,
+        passedUsers: questionaire.passedUsers,
+        passScore: questionaire.passScore,
+      };
+    });
```

Comment:
As far as I can tell it's not necessary to map in this case, I could be mistaken but if you agree we can remove this and just return allQuestionnaires instead of mappedQuestionnaries in the body